tests/e2e/scale/test_pv_scale_and_respin_ceph_pods.py

```diff
@@ -132,6 +133,15 @@ def cleanup(self):
         pytest.param(*["mon"], marks=[pytest.mark.polarion_id("OCS-764")]),
         pytest.param(*["osd"], marks=[pytest.mark.polarion_id("OCS-765")]),
         pytest.param(*["mds"], marks=[pytest.mark.polarion_id("OCS-613")]),
+        pytest.param(
+            *["cephfsplugin_provisioner"], marks=[pytest.mark.polarion_id("OCS-2641")]
+        ),
+        pytest.param(
+            *["rbdplugin_provisioner"], marks=[pytest.mark.polarion_id("OCS-2639")]
+        ),
+        pytest.param(*["rbdplugin"], marks=[pytest.mark.polarion_id("OCS-2643")]),
+        pytest.param(*["operator"], marks=[pytest.mark.polarion_id("OCS-2640")]),
```

Comment:
operator restart might not be relevant here.

Reply:
Removed operator resource.